lib/models/blueprint.js

```diff
@@ -385,14 +385,26 @@ let Blueprint = CoreObject.extend({
   /**
     @private
     @method _checkInRepoAddonExists
-    @param {String} inRepoAddon
+    @param {Object} options
   */
-  _checkInRepoAddonExists(inRepoAddon) {
-    if (inRepoAddon) {
-      let addon = findAddonByName(this.project, inRepoAddon);
+  _checkInRepoAddonExists(options) {
+    let addon;
+
+    if (options.inRepoAddon) {
+      addon = findAddonByName(this.project, options.inRepoAddon);
+
       if (!addon) {
-        throw new SilentError(`You specified the in-repo-addon flag, but the ` +
-          `in-repo-addon '${inRepoAddon}' does not exist. Please check the name and try again.`);
+        throw new SilentError(`You specified the 'in-repo-addon' flag, but the ` +
+          `in-repo-addon '${options.inRepoAddon}' does not exist. Please check the name and try again.`);
+      }
+    }
+
+    if (options.in) {
+      addon = findAddonByPath(this.project, options.in);
+
+      if (!addon) {
+        throw new SilentError(`You specified the 'in' flag, but the ` +
+          `in-repo-addon '${options.in}' does not exist. Please check the name and try again.`);
```

Comment:
Should probably not say `in-repo-addon` here

Reply:
Yep, agree. I was thinking just 'in repo addon', not dashes. Would that be more clear?